### DISCUSSION TITLE
mongoose-release-bot has likely wrongly mentioned a user in a different place

### DISCUSSION THREAD

**Discussion by hasezoey** [Automattic/mongoose#14472] [General]
i have noticed that for the release of 8.2.4, the release notes were:

---
8.2.4 / 2024-03-28
==================
 * types(query): bring "getFilter" and "getQuery" in-line with "find" and other types #14463 [noseworthy](https://github.com/noseworthy)
 * types(schema): re-export the defintion for SearchIndexDescription #14464
 * docs: removed unused hook from docs #14461 [bernardarhia](https://github.com/bernardarhia)
---

where it (to me seemingly wrongly) assigned user `noseworthy` to `#14463`, but the users PR is actually the second line `#14464`.

i dont know where to properly report this bug for the bot, so i opened it here.

@vkarpov15

**Comment by vkarpov15** [comment]
@hasezoey thanks for noticing, fixed in https://github.com/Automattic/mongoose/commit/b4312696e0fb5d133dfdb82e42afe39409b1659d and https://github.com/Automattic/mongoose/releases/tag/8.2.4
